var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -282,6 +285,13 @@ class Llvm(CMakePackage, CudaPackage):
     # https://spack.readthedocs.io/en/latest/packaging_guide.html#making-a-package-discoverable-with-spack-external-find
     executables = ['clang', 'flang', 'ld.lld', 'lldb']
 
+    def patch(self):
```

Comment:
This looks like it should have an @when decorator on it. What version of llvm/compiler-rt is this for?

Reply:
Added and builds have succeeded.